### ISSUE TITLE
after npm install, it throw error when running npm start

### ISSUE THREAD

**Issue by carlguo2016** [blacksonic/angular2-aot-webpack#40]
after npm install, it throw error when running npm start

**Comment by sonicoder86** [comment]
@carlguo2016 Can you share details of your setup? Error message, operating system, node version etc.

**Comment by sonicoder86** [comment]
Had an issue with wrong ```zone.js``` version, did it fix the issue?

**Comment by ajziehl** [comment]
Not sure if he's getting the same error as me, but I am getting an error after doing these 3 steps:

```
git clone https://github.com/blacksonic/angular2-aot-webpack.git .
npm i
npm start
```

Giant stack trace in console, here's part of it:
```
TypeError: Cannot read property 'exclude' of undefined
    at applyDefaults (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\instance.ts:266:72)
    at Object.ensureInstance (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\instance.ts:145:5)
    at compiler (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\index.ts:37:20)
    at Object.loader (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\index.ts:18:18)
    at LOADER_EXECUTION (C:\Users\User\temp\angular2-aot-webpack\node_modules\loader-runner\lib\LoaderRunner.js:114:14)

ERROR in ./src/app/bootstrap.ts
Module build failed: TypeError: Cannot read property 'exclude' of undefined
    at applyDefaults (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\instance.ts:266:72)
    at Object.ensureInstance (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\instance.ts:145:5)
    at compiler (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\index.ts:37:20)
    at Object.loader (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\index.ts:18:18)
 @ multi bootstrap

ERROR in ./src/app/bootstrap.aot.ts
Module build failed: TypeError: Cannot read property 'exclude' of undefined
    at applyDefaults (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\instance.ts:266:72)
    at Object.ensureInstance (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\instance.ts:145:5)
    at compiler (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\index.ts:37:20)
    at Object.loader (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\index.ts:18:18)
 @ multi bootstrap.aot

ERROR in ./src/app/polyfill.ts
Module build failed: TypeError: Cannot read property 'exclude' of undefined
    at applyDefaults (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\instance.ts:266:72)
    at Object.ensureInstance (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\instance.ts:145:5)
    at compiler (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\index.ts:37:20)
    at Object.loader (C:\Users\User\temp\angular2-aot-webpack\node_modules\awesome-typescript-loader\src\index.ts:18:18)
 @ multi polyfill
```

Seems to be an issue with TypeScript 2.1.4 and awesome-typescript-loader. The fix for me was to use TypeScript 2.0.10.

**Comment by sonicoder86** [comment]
Upgrading ```awesome-typescript-loader``` to beta solves the issue also.
